api/working-with-extensions/continuous-integration.md

```diff
@@ -95,3 +95,59 @@ script:
     yarn && yarn compile && yarn test
 cache: yarn
 ```
+
+## Automated publishing
+
+You can configure the CI to publish a new version of the extension automatically.
+
+The publish command is similar to the command of publishing from local environment using [`vsce`](https://github.com/Microsoft/vsce) service but this time the command needs to pass also the the Personal Access Token ("PAT").
+
+So, to configure the CI to also publish the extension, follow those steps:
+
+1. Install `vsce` as `devDependencies` (`npm install vsce --save-dev` or `yarn add vsce --dev`)
+2. Add a `deploy` script in `package.json`
+
+```json
+"scripts": {
+  "deploy": "vsce publish -p [your-personal-access-token]"
```

Comment:
Not sure we should advocate for doing this. I would recommend simply merging these two sections and skip this step altogether. Using secret variables is the only right way to do this.